web/docs/auth/migrate-from-11.md

```diff
@@ -86,165 +142,311 @@ You can read more about the new auth system in the [Auth Entities](./entities) s
 
 **We'll put extra info for migrating a deployed app in a box like this one.**
 :::
 
-### 1. Migrate to the New Auth System
+#### 1. Migrate to the New Auth System
 
 You can follow these steps to migrate to the new auth system:
 
 1. Upgrade Wasp to the latest 0.12 version.
 
-  These instructions are for migrating the auth from Wasp `0.11.X` to Wasp `0.12.X`, which means that they work for example for both `0.12.0` and `0.12.5` versions. We suggest that you install the latest 0.12 version of Wasp. Find the available Wasp versions in the [Releases](https://github.com/wasp-lang/wasp/releases) section of our GitHub repo.
+These instructions are for migrating the auth from Wasp `0.11.X` to Wasp `0.12.X`, which means that they work for example for both `0.12.0` and `0.12.5` versions. We suggest that you install the latest 0.12 version of Wasp. Find the available Wasp versions in the [Releases](https://github.com/wasp-lang/wasp/releases) section of our GitHub repo.
 
-  Then you can install that version with:
+Then you can install that version with:
 
-  ```bash
-  curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s -- -v 0.12.0
-  ```
+```bash
+curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s -- -v 0.12.0
+```
 
   <small>
 
-  In the above command, replace `0.12.0` with the version you want to install.
+In the above command, replace `0.12.0` with the version you want to install.
 
   </small>
 
 1. Bump the version to `^0.12.0` in `main.wasp`.
 1. Create the new auth tables in the database by running:
 
-  ```bash
-  wasp db migrate-dev
-  ```
+```bash
+wasp db migrate-dev
+```
 
-  You should see the new `Auth`, `AuthIdentity` and `Session` tables in your database. You can use the `wasp db studio` command to open the database in a GUI and verify that the tables are there.
+You should see the new `Auth`, `AuthIdentity` and `Session` tables in your database. You can use the `wasp db studio` command to open the database in a GUI and verify that the tables are there.
 
 1. Write your data migration function(s) in `src/server/migrateToNewAuth.ts`
-    - In the previous step, we migrated the schema, and now we need to prepare logic for migrating the data.
-    - Below you can find [examples of migration functions](#example-migration-scripts) for each of the auth methods. They should be fine to use as-is, meaning you can just copy them, but you can also modify them to your needs. You will want to have one function per each auth method that you use in your app.
+   - In the previous step, we migrated the schema, and now we need to prepare logic for migrating the data.
+   - Below you can find [examples of migration functions](#example-migration-functions) for each of the auth methods. They should be fine to use as-is, meaning you can just copy them, but you can also modify them to your needs. You will want to have one function per each auth method that you use in your app.
 1. Add the migration function(s) to the `db.seeds` config:
```

Comment:
I've modified the imports and paths in this file in this PR: https://github.com/wasp-lang/wasp/pull/1723/files#diff-6abd9f18e900b4b6b6d7597c73fb4c4ed1f99cdc1344ea5df6272b7477fa76bfR131